### DISCUSSION TITLE
DAX Studio on Windows Terminal Server support

### DISCUSSION THREAD

**Discussion by spotpuff** [DaxStudio/DaxStudio#591] [Q&A] [answered]
I was wondering if DAX studio is compatible with Windows Server 2016 with multiple user sessions. I have only ever used the desktop client on WIndows 10.

Are there any special requirements for server versions of Windows?

**Comment by dgosbell** [comment, marked as answer]
Yes, it should work fine on Windows Server 2016, I think I've even run it on 2012R2. As long as the server has .net 4.7.2 or later it should work fine. And the installer should check for that and offer to install it if it finds the server does not meet that requirement. There is no special requirements or server version.
